main.py

```
@@ -0,0 +1,167 @@
+"""Application entry point — bootstrap, wire, run."""
+
+from __future__ import annotations
+
+import asyncio
+import json
+import logging
+import signal
+import sys
```

Flake8 (7.3.0)

[error] 'sys' imported but unused

(F401)